docs/content/preview/yugabyte-platform/create-deployments/connect-to-universe.md

```diff
@@ -21,13 +21,36 @@ You can connect to the database on a universe in the following ways:
 
 ## Download the universe certificate
 
-If the universe uses encryption in transit, to connect you need to first download the universe TLS root certificate. Do the following:
+If the universe uses Client-to-Node encryption in transit, to connect you need to first download the universe TLS certificate. Do the following:
 
 1. Navigate to **Configs > Security > Encryption in Transit**.
 
-1. Find the certificate for your universe in the list and click **Actions** and download the certificate.
+1. Find your universe in the list.
 
-For more information on connecting to TLS-enabled universes, refer to [Connect to clusters](../../security/enable-encryption-in-transit/#connect-to-clusters).
+1. Download the certificate.
+
+    - If you are connecting using a YSQL client (such as ysqlsh), click **Actions**, and choose **Download YSQL Cert**.
```

Comment:
```suggestion
    - If you are connecting using a YSQL client (such as ysqlsh) and wish to verify the server's identity at the client, click **Actions**, and choose **Download Root CA Cert**. Supply this CA cert to the `ysqlsh` command in the `sslrootca` parameter as [shown here](https://www.yugabyte.com/blog/securing-yugabytedb-client-to-server-encryption/#verification-of-server-certificates) to enable the ysqlsh client to verify the server certificate.
```

The information here needs a bit of a rewrite. The full story is described here - https://www.yugabyte.com/blog/securing-yugabytedb-client-to-server-encryption/#verification-of-server-certificates

In general, for YSQL clients, the user only needs the CA cert, similar to YSQL. The "YSQL cert" is only required if they have enabled postgres certificate auth as described in https://www.yugabyte.com/blog/securing-yugabytedb-client-to-server-encryption/#verification-of-client-certificates but this is not very common and it requires the user to perform additional steps to enable such auth. I think we can ignore this, if we want we can condense this section ("verification of client certificates") here.